### ISSUE TITLE
[Bug] Unable to delete a line entirely using CMD+DEL

### ISSUE THREAD

**Issue by henri9813** [AppFlowy-IO/AppFlowy#6885]
### Bug Description

Usually, CMD+ DEL ( in macOS ) permit to delete the whole line.

It doesn't work in appflowy. ( it does nothing ).

### How to Reproduce

Write a line, and press CMD+ DEL

### Expected Behavior

It should work.

### Operating System

MacOS

### AppFlowy Version(s)

0.7.5

**Comment by LucasXu0** [comment]
@henri9813 We support CMD+Backspace to delete a line. Is it common on your keyboard to use CMD+Delete to delete a line? I tested on macOS, and most applications use CMD+Backspace to delete a line.

**Comment by henri9813** [comment]
Sorry, I mean cmd + backspace, but it works randomly.

**Comment by henri9813** [comment]
I dont know precisely box reproduce it, but it happens often on the first block of a new card page

**Comment by LucasXu0** [comment]
@henri9813 Do you mean the document on the board card?

<img width="978" alt="Screenshot 2024-12-03 at 14 20 24" src="https://github.com/user-attachments/assets/785947ba-194e-42ef-9f49-afa8e049a754">

**Comment by henri9813** [comment]
Yes, often, the first line of the first black don't support cmd + back neither other shortcut

**Comment by henri9813** [comment]
but it's random 🤡

**Comment by LucasXu0** [comment]
@henri9813 I still haven't been able to reproduce it on macOS. I can always use cmd+backspace/del to remove the entire line. If you have any more reproducible steps, that would be better to address this issue.
